app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -35,9 +35,8 @@
         {% include "toolkit/page-heading.html" %}
     {% endwith %}
     <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-
-    <p><strong>Suppliers will be able to apply for:</strong></p>
-    <p class="padding-bottom-small">2 weeks ending at midnight.</p>
+    <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
+    <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date }}.</p>
 
     <p><strong>Supplier questions will be sent to:</strong></p>
```

Comment:
I wasn't putting in dmspeak at currently, as it still hasn't really got enough of an explanation to make sense as a class or examples in the toolkit.  It will also have to be changed to work for this example.  I wanted to add this later on when we weren't stuck for time.

I thought the email being on a separate line was a design decision, but I agree that it does seem weird and will change it.